cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -429,6 +430,7 @@ func (dp *devicePlugin) scan() (dpapi.DeviceTree, error) {
 		}
 
 		isPFwithVFs := pluginutils.IsSriovPFwithVFs(path.Join(dp.sysfsDir, f.Name()))
+		tileCount = max(labeler.GetTileCount(dp.sysfsDir, f.Name()), tileCount)
```

Comment:
Shouldn't this rather check whether tile count differs between GPUs and log warning about unsupported HW configuration, if it does?

Reply:
Yep, it could. The "problem" is that it would only stay in the logs. User wouldn't know about it unless he/she checks specific GPU plugin logs.

Reply:
If no tile resource is provided for such a broken node setup, user can notice something being wrong, and hopefully checks the log.

Reply:
I added a warning.